common/templates/common/sample_django_react.html

```diff
@@ -4,10 +4,10 @@
 
 {% extends 'base_templates/base.html' %}
 {% load django_react_components %}
-{% load render_bundle from webpack_loader %}
+{% load render_bundle webpack_static from webpack_loader %}
 
 {% block body %}
-  {% render_bundle 'runtime' %}
+  <script src="{% webpack_static 'runtime.js' %}"></script>
```

Comment:
This is a breaking change, which is not ideal since the upgrade of `webpack-bundle-tracker` doesn't really add any benefit that we're using. Noah and I agree that generally being on the latest (LTS) version of software is a good thing, but since the current implementation of djangre isn't great anyway, we're going to eventually make larger breaking changes in the future anyway, making this particular upgrade not worth doing.

There is still an issue, though, that the version of `django-webpack-loader` in `requirements.in` is not locked, and so new apps created with https://github.com/zagaran/django-template will break, so the solution we agreed on is to lock the version for `django-webpack-loader` instead.